chatminer/chatparsers.py

```diff
@@ -161,10 +163,10 @@ def _infer_datatime_format(self):
     def _parse_message(self, mess):
         if self._datetime_format in (StartOfDateType.DAY,
                                      StartOfDateType.AMBIGUOUS):
-            datetime = datetimeparser.parse(mess.split('-', 1)[0],
+            time = datetimeparser.parse(mess.split('-', 1)[0],
```

Comment:
'had' to change this variable naming due to importing datetime package to parse the date_unixtime from the telegram json